libs/react-components/src/tedi/components/cards/card/card-stories-templates.tsx

```diff
@@ -0,0 +1,546 @@
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Text modifiers="bold">Title</Text>
+            <Text color="secondary">Description</Text>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Text modifiers="bold">Title</Text>
+            <Row>
+              <Col>
+                <Text color="secondary">Description</Text>
+              </Col>
+              <Col width="auto">
+                <StatusBadge color="brand">Approved</StatusBadge>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col className="display-flex align-items-center gap-3">
+                <Icon name="monitor_heart" />
+                <Text color="secondary">Description</Text>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col className="display-flex align-items-center gap-3">
+                <Icon name="monitor_heart" />
+                <VerticalSpacing size={0}>
+                  <Text modifiers="bold">Title</Text>
+                  <Text color="secondary">Description</Text>
+                </VerticalSpacing>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col className="display-flex align-items-center gap-3">
+                <Icon name="monitor_heart" />
+                <VerticalSpacing size={0}>
+                  <Text modifiers="bold">Title</Text>
+                  <Text color="secondary">Description</Text>
+                </VerticalSpacing>
+              </Col>
+              <Col width="auto" className="display-flex align-items-center">
+                <Button>Create</Button>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col lg={6} sm={12}>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col>
+                <Text modifiers="bold">Title</Text>
+                <Text color="secondary">Description</Text>
+                <Separator spacing={1.5} />
+                <Row justifyContent="center">
+                  <Col width="auto">
+                    <Button>Create</Button>
+                  </Col>
+                </Row>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+  </VerticalSpacing>
+);
+
+export const CardInfoTemplate: StoryFn<CardStory> = () => (
```

Comment:
All the repeating code would be nice to be removed and moved into functions, that take needed properties inside.

Reply:
I replaced the rows/cols with VerticalSpacing and while I agree, it's an ugly long code but since it's in a separate file like (card-stories-templates) then maybe it's useful for someone to copy that template into their project? It's only descriptive file with examples

Reply:
Copy from directly code? Because the template is not shown in Storybook currently.
![image](https://github.com/user-attachments/assets/dac25a4a-f416-4513-a418-78a1eaf6d236)

To copy the code, the itself doenst have to be written in that way. For example take community/Icon stories -> It has Templates that include functions and passing props, but in storybook the final React code is shown:
![image](https://github.com/user-attachments/assets/5d9dcd69-cbb2-4e54-bfb1-8ae05de6495f)